### PR TITLE
support php 8.0

### PR DESCRIPTION
**Pull request by benharri**

### CODE REVIEW

**Comment by mnapoli** [comment]
Thanks!

Because of the PHPUnit upgrade, the build is failing on PHP 7.2.

I'm ok to drop support for PHP 7.2, could you update the PR with that? Also could you make tests run on PHP 8 on Travis CI?

Thanks!

**Comment by benharri** [comment]
Sounds good. 7.2 will be out of security support starting Monday so that makes sense. Looks like travis-ci doesn't have a php8 image yet... that download is 404ing.

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 language: php
 
 php:
-  - 7.2
   - 7.3
   - 7.4
+  - 8.0
```
Comment (mnapoli) [inline .travis.yml:6]:
```suggestion
  - 8.0snapshot
```

`8.0` doesn't exist in Travis yet.

**Comment by benharri** [comment]
Thanks for finding the correct version tag - I couldn't find that for some reason in my own googling

**Pull request merged**